src/browser/web-view.js

```diff
@@ -751,6 +764,10 @@ const viewFrame = (model, address) =>
     onBlur: on(address, always(BlurShell)),
     onFocus: on(address, always(FocusShell)),
     // onMozbrowserAsyncScroll: on(address, decodeAsyncScroll),
+    // @TODO send force changed actions and update webviews animation code
+    // to respond live 1:1 with push.
+    // onServoMouseForceChanged: on(address, decodeForceChanged),
+    onServoMouseForceDown: on(address, decodeForceDown),
```

Comment:
I assumed that to get correct event capture from moziframe, we need to bind event handler to moziframe. Otherwise the page can capture events before they bubble. I believe the way it works is that iframe gets first dibs on the event before page can cancel, then it bubbles up. Please correct me if I'm wrong here.